kolibri/plugins/learn/assets/src/views/LibraryPage/index.vue

```diff
@@ -260,23 +260,17 @@
         searchMore,
         removeFilterTag,
         clearSearch,
-        setCategory,
         currentRoute,
       } = useSearch();
       search();
-      const {
-        resumableContentNodes,
-        moreResumableContentNodes,
-        fetchResumableContentNodes,
-        fetchMoreResumableContentNodes,
-      } = useLearnerResources();
+      const { fetchResumableContentNodes } = useLearnerResources();
```

Comment:
We could defer even this loading just to the child component.

Reply:
meaning the card grid?

Reply:
Yes, `useLearnerResources` is used extensively inside the ResumableCardGrid, so it could just handle all of this!